modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-static-network_{context}"]
+= Configuring static networking
+
+If you install {op-system-first} from an ISO image, you can add kernel arguments
+when you boot that image to configure the node's networking.
+The following table describes and illustrates how to use those kernel arguments.
+
+.Configuring static networking
```

Comment:
Because the bonding examples show DHCP and static IP assignment, I think this section should be renamed.

Perhaps `Configuring Advanced Networking`

Reply:
I agree. That name would also allow us to expand the content going forward for all things networking. @kalexand-rh I wanted to give you a heads up that we are changing the name of the heading and table.

Reply:
I've been told that gerunds should be limited to procedures, when possible. So I'm going to change the table and section name to "Configure advanced networking".